minigui/fetch-and-run.sh

```diff
@@ -48,41 +50,56 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
   echo "Downloading Model files:"
   echo "--------------------------------------------------"
 
-  MODEL_SUFFIXES=( "data-00000-of-00001" "index" "meta" )
-  mkdir -p $MINIGUI_MODEL_TMPDIR
+  model_tmpdir="${MINIGUI_TMPDIR}/models"
+  control_tmpdir="${MINIGUI_TMPDIR}/control"
+  mkdir -p ${model_tmpdir}
+  mkdir -p ${control_tmpdir}
 
+  MODEL_SUFFIXES=( "data-00000-of-00001" "index" "meta" )
   for suffix in "${MODEL_SUFFIXES[@]}"
   do
-    file_to_check="${MINIGUI_MODEL_TMPDIR}/${MINIGUI_MODEL}.${suffix}"
+    file_to_check="${model_tmpdir}/${MINIGUI_MODEL}.${suffix}"
     echo "Checking for: ${file_to_check}"
     if [[ ! -f "${file_to_check}" ]]; then
-      gsutil cp gs://${MINIGUI_BUCKET_NAME}/${MINIGUI_GCS_DIR}/${MINIGUI_MODEL}.${suffix} $MINIGUI_MODEL_TMPDIR/
+      gsutil cp gs://${MINIGUI_BUCKET_NAME}/${MINIGUI_GCS_DIR}/${MINIGUI_MODEL}.${suffix} ${model_tmpdir}/
     fi
   done
 
   # Assume models need to be frozen if .pb doesn't exist.
```

Comment:
if we use anything past v10, the .pb's exist in minigo-pub already

Reply:
as a bonus, that means we don't have to guess at the conv-width param (likely source of bugs)

Reply:
We still need conv-width to run inference though